docs/trulens/api/database/migration.md

```diff
@@ -29,7 +29,7 @@ blank.
 
 If you need to change this prefix after migration, you may need to specify the
 old prefix when invoking
-[migrate_database][trulens_eval.tru.Tru.migrate_database]:
+[migrate_database][src.core.trulens.core.tru.Tru.migrate_database]:
```

Comment:
Is this (and the other changed references) supposed to be a filepath or a python package path? If the latter, the python package import would just be `trulens.core.tru.Tru.migrate_database` (omitting the first src.core since that's not used here).